R/doc-params.R

```diff
@@ -0,0 +1,54 @@
+#' Argument type: tidy-select
+#'
+#' @description
+#' This page the describes the `<tidy-select>` argument modifier which indicates
+#' the argument uses tidy evaluation with **tidy selection**. Tidy selection
+#' provides a concise DSL for selecting variables based on their names.
+#' If you've never head of tidy evaluation before, start with
+#' `vignette("programming")`.
+#'
+#' # General usage
+#'
+#' If you have a data frame with variables `apple`, `banana`, `cantelope`,
```

Comment:
```suggestion
#' If you have a data frame with variables `apple`, `banana`, `cantaloupe`,
```
There are cantaloupes 🍈 and then there are antelopes 🦌